app/shared/reducers/validate.js

```diff
@@ -10,14 +10,36 @@ export default function validate(state = {}, action) {
     return {};
   }
 
-  const matches = /^VALIDATE_(.*)_(PENDING|SUCCESS|FAILURE|CONFIRMING|NULL)$/.exec(type);
+  const matches = /^VALIDATE_(.*)_(PENDING|SUCCESS|FAILURE|NULL)$/.exec(type);
```

Comment:
The `NULL` here should be removed once validation is completely removed for staking.

Reply:
I have been using it to reset the VALIDATE_STAKE_.. state variables, when someone  opens the modal for the second time. What do you recommend doing instead?

Reply:
Use the `clearSystemState()` call from the `transfer.js` actions (we should find a better home for that).